app/styles/_core.less

```diff
@@ -1235,3 +1235,8 @@ pre.clipped {
     width: 100%;
   }
 }
+
+// Override Patternfly z-index for toast notifications list to keep toasts above modals
+.toast-notifications-list-pf {
+  z-index: @zindex-modal + 5;
```

Comment:
Curious if `+ 5` is arbitrary or if we're using `+ 1`, `+ 2`, `+ 3` ... elsewhere?